lib/themes/dosomething/paraneue_dosomething/js/components/carousel.js

```diff
@@ -0,0 +1,30 @@
+const React = require('react');
+
+import ContextSlide from './context_slide.js';
+
+const Carousel = React.createClass({
+  getInitialState: function() {
+    return {
+      activeSlide: 1,
+    };
+  },
+  render: function() {
+    var slide;
+    switch (this.state.activeSlide) {
+      case 1: slide = <ContextSlide />; break;
```

Comment:
Ah so pass the slides in as a property & keep activeSlide on the state, then do something like `this.props.slides[this.state.activeSlide]`